### ISSUE TITLE
Lesson 11 mouse inversion

### ISSUE THREAD

**Issue by parente** [senchalabs/philogl#2]
This isn't anything critical, but I figured I'd report it at least.

Compare the effect click-and-drag has on the original Learning WebGL demo with PhiloGL:

http://learningwebgl.com/lessons/lesson11/index.html
http://senchalabs.github.com/philogl/PhiloGL/examples/lessons/11/

1) Rotate the moon 90 degrees about its initial vertical axis.
2) Click-and-drag up and down.

In the original, the up-and-down motion rotates the moon about the viewport horizontal axis. In the PhiloGL demo, the up-down drag rotates about the original model horizontal axis (whic is no longer aligned with the viewport).

**Comment by philogb** [comment]
Yes, I'm thinking that besides the update method in O3D.Model (that only updates the matrices for the model leaving all vertices on their original position) there should be a updateComponents (or something like that) that would leave the matrices as-is but would update all vertices and normals for the object.

Thanks for reporting this :)
